Lib/_pydecimal.py

```diff
@@ -5931,7 +5931,7 @@ def _dpower(xc, xe, yc, ye, p):
     if pc == 0:
         # we prefer a result that isn't exactly 1; this makes it
         # easier to compute a correctly rounded result in __pow__
-        if ((len(str(xc)) + xe >= 1) == (yc > 0)): # if x**y > 1:
+        if (len(str(xc)) + xe >= 1) == (yc > 0): # if x**y > 1:
```

Comment:
It was not a typo.  Lets not touch code without good reasons.

Reply:
I changed it back however should we not be going for consistency? None of the other if statements in the file feature these redundant braces.